scripts/build.bash.bats

```diff
@@ -48,11 +52,11 @@ setup() {
 	./dist/blargles
 
 	# Assert the zip was created.
-	assert_file_exists "zip/blargles.zip"
+	assert_file_exists "out/blargles.zip"
```

Comment:
Why is the output directory changing here?

Reply:
Previously the test was using a hard-coded dir `zip` but now it's relying on the default dir which is `out`. (It should never have been `zip` really, since the convention is to use `out`.)